README.md

```diff
@@ -1 +1,77 @@
-# nvim-package-specification
\ No newline at end of file
+# Plugin metadata specification
+
+The neovim package specification supports a single, top-level package metadata file. This file can be *either* 'plugin.lua' or 'plugin.json'. The file is a superset of the Rockspec File Format and can contain the following fields
```

Comment:
It's a superset. Build keys might be useful for things like telescope-fzf-native (that's all I can think of)

Reply:
Sorry: my confusion is that Rockspec supports some keys (such as build, differentiated dependency types, etc.) that this does not seem to (see https://github.com/luarocks/luarocks/wiki/Rockspec-format). I'm not suggesting that we should include those keys, but rather that this format is a subset of the Rockspec format (by keys supported).

Reply:
Yeah, this version is both a sub and superset haha (in a way) since it has additional keys to get around not using luarocks, but not all build steps are going to be supported, AFAIK build has some way of specifying the C build steps for luarocks.

Reply:
At some point in the future, I would love us to better support plugins that have C sources, just like `telescope-fzf-native`. `nvim-treesitter` may be able to leverage this somehow too which needs to compile parsers?